docusaurus/docs/React/contexts/component-context.mdx

```diff
@@ -171,6 +171,14 @@ Custom UI component for a deleted message.
 | --------- | ---------------------------------------------------------------------------------------------------------------------- |
 | component | [MessageDeleted](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Message/MessageDeleted.tsx) |
 
+### MessageListNotifications
+
+Custom UI component that displays message and connection status notifications in the `MessageList`.
+
+| Type      | Default                                                                                                                                  |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| component | [DefaultMessageListNotifications](https://github.com/GetStream/stream-chat-react/blob/master/src/components/MessageList/MessageList.tsx) |
+
```

Comment:
why the spacing look weird on the borders in the mdx files?

Reply:
I don't know, but it looks fine when running and also doesn't seem to have spaces in the file...